ui/src/global.css

```diff
@@ -3,3 +3,34 @@ body,
 html {
     height: 100%;
 }
+
+/* Allows the tag Chips to wrap lines */
+.MuiChip-label {
+    white-space: normal !important;
+}
+
+/* The wrapping box around the input and icon for the datetime picker */
+.MuiInputBase-root.MuiInput-root.MuiInputBase-formControl.MuiInput-formControl.MuiInputBase-adornedEnd {
```

Comment:
Add this only to the DateTimeSelector so it doesn't break other text fields.

Reply:
I have added a class to the parent `time-picker` and made all that css specific to that.